### ISSUE TITLE
Es plugin as output for Humio stopped working after updating fluent bit version 1.8.4

### ISSUE THREAD

**Issue by surajthakur** [fluent/fluent-bit#3986]
Hi Team, 

We updated fluent-bit version from 1.7.4 to 1.8.4( latest coming from apt repo).  After the update the agent start giving error
```
[2021/08/23 17:05:28] [ warn] [engine] failed to flush chunk '9909-1629702310.123277989.flb', retry in 9 seconds: task_id=0, input=storage_backlog.2 > output=es.0 (out_id=0)
[2021/08/23 17:05:37] [error] [output:es:es.0] HTTP status=400 URI=/_bulk, response:
Humio can only handle index actions for the elastic search api.
``` 
We are running OS - Debian 9 
 
The fluentbit config: output plugin config is  as below
```
[OUTPUT]
        Name        es
        Match       *
        Host        humio-ip
        Port        9200
        Index       <humio_repo_name>
        HTTP_User   <ingest-key>
```

Any ideas what could be wrong ? 
We have rolled back to 1.7.4 and now it's working fine. 
It was hard to find old version of fluentbit in apt repo neither the documentation gives any link of direct download. 
From some github issue i was able to get `https://packages.fluentbit.io/debian/stretch/pool/main/t/td-agent-bit/td-agent-bit_1.7.4_amd64.deb` 
Is there any docs for correct direct download link and format of the url  

Thanks

**Comment by joenzx** [comment]
I have encountered this error, it is a problem with es, check whether your es cluster is normal @surajthakur

**Comment by surajthakur** [comment]
> I have encountered this error, it is a problem with es, check whether your es cluster is normal @surajthakur

@joenzx , but how is it working fine by downgrading the version ?

**Comment by Crevil** [comment]
We see the same. Upgrading from 1.7.9 to 1.8.0 breaks ingestion in Humio. (Same with versions up until 1.8.4)

The only change I can see that _might_ be related is https://github.com/fluent/fluent-bit/pull/3788 which touches the `es` output.

**Comment by Crevil** [comment]
That indeed looks to be the reason.

The reasoning for making that change makes great sense I think, so I guess we should direct this issue to Humio and their compatible elasticsearch API instead. I'll open an issue with them to get their input.

**Comment by Crevil** [comment]
I have spoken to Humio support and they have provided the following answer. Slightly redacted.

```
We've gone ahead and created a feature request to let our engineering team know about this issue with ingesting data from the elasticsearch output plugin on fluent-bit 1.8+. I've also notified them that this is a priority for you as it is preventing fluent-bit upgrades. If you're able to keep an eye on our Release Notes page, any updates to this issue will be shown there:
https://library.humio.com/notices/releases/release-notes/#

One thing I did want to mention is that it might be possible to downgrade the elasticsearch plugin itself after upgrading fluent-bit. I'm not positive how that would work but it may be worth looking into as a workaround in the meantime. I'll go ahead and close this ticket out for now but feel free to reach out if you have any further questions!
```

I'm unsure about their proposal to downgrade the elasticsearch plugin. Does any one know if that is doable at all?

**Comment by surajthakur** [comment]
@Crevil, From some github issue i was able to get https://packages.fluentbit.io/debian/stretch/pool/main/t/td-agent-bit/td-agent-bit_1.7.4_amd64.deb
Is there any docs for correct direct download link and general format of the url  ? 

Thanks

**Comment by Crevil** [comment]
Just got another update from Humio

```
Good news! I heard back on the feature request and it looks like they've already come up with a potential solution for this. I can't give an exact timeline on when that would be released since they'll likely want to do some testing/verification but I wouldn't expect it to be too far out. (...)
```

We are running the same version of Humio currently.

I cannot help with the download URLs as we are using the Docker images of fluent-bit.

**Comment by surajthakur** [comment]
Thanks for the update @Crevil , appreciate that. 
We were also running using the docker images, but we also wanted to use fluenbit to collect the host based logs. So we had to install this on host as td-agent.

**Comment by Crevil** [comment]
This should be fixed in Humio 1.31.0. We have yet to test it out though.

**Comment by surajthakur** [comment]
Thanks for the update @Crevil. Looking forward to know your results.
